src/main/java/lotto/controller/LottoGame.java

```diff
@@ -0,0 +1,26 @@
+package lotto.controller;
+
+import lotto.domain.LottoBuy;
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoTicket;
+import lotto.domain.Money;
+import lotto.domain.WinningLotto;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoGame {
+
+    public static void playLotto() {
+        Money money = new Money(InputView.getMoney());
+        LottoBuy lottoBuy = new LottoBuy(money);
+        LottoTicket lottoTicket = lottoBuy.buyAutoLottos(lottoBuy.getCount());
+
+        ResultView.printCount(lottoBuy.getCount());
+        ResultView.printLottos(lottoTicket);
```

Comment:
```suggestion
        ResultView.printLottos(lottoTicket);
```
`lottoBuy.getCount()`는 외부에 노출하지 않도록 수정해 보는 것은 어떨까요?